Vault-Deployment/values/vault-values.yaml

```diff
@@ -46,53 +44,37 @@ server:
     name: hashicorp-vault-license
     readOnly: true
 
-
-
-
-  standalone:
-    enabled: false
-  affinity: |
-    podAntiAffinity:
-      # Choose a soft scheduling preference, because we distribute 3-5 Pods to 3 nodes/zone:
-      # https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node/#an-example-of-a-pod-that-uses-pod-affinity
-      #
-      # The anti-affinity rule says that we prefer to schedule the Vault server pods (identified by name/instance labels)
-      # in separate zones (topologyKey kubernetes.io/zone).
-      preferredDuringSchedulingIgnoredDuringExecution:
-        - weight: 100
-          podAffinityTerm:
-            labelSelector:
-              matchLabels:
-                app.kubernetes.io/name: {{ template "vault.name" . }}
-                app.kubernetes.io/instance: "{{ .Release.Name }}"
-                component: server
-            topologyKey: kubernetes.io/zone
-
-
+  ingress:
+    enabled: true
+    annotations:
+      nginx.ingress.kubernetes.io/ssl-passthrough: "true"
+    ingressClassName: "nginx"
+    pathType: Prefix
+    activeService: true
+    hosts:
+      - host: vault.playground.lab
+        paths: []
 
   ha:
     enabled: true
-    replicas: 3
+    # can be scaled to 3
```

Comment:
@Xelef2000 let's keep it simple here, all right? I adjusted the instructions in the README accordingly.